source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc

```diff
@@ -16,7 +16,18 @@ The order of the options or settings shown in this section may differ depending
      -e '/^OVESETUP_CONFIG\/fqdn/d' \
      /etc/ovirt-engine-setup.conf.d/20-setup-ovirt-post.conf
 ----
-
+. Remove the apache/grafana PKI files, so that they are regenerated by `engine-setup`, if enabled, using correct values. 
```

Comment:
It's not clear what "if enabled" means in this context. Do you need this phrase?

Reply:
'enabled' means that the user replied 'yes' when asked whether to configure apache/grafana.